tools/gce/linux_performance_worker_init.sh

```diff
@@ -135,6 +135,12 @@ curl -sSL https://get.rvm.io | bash -s stable --ruby
 source ~/.rvm/scripts/rvm
 gem install bundler
 
+# PHP dependencies
```

Comment:
Is this already installed on the workers?

Reply:
Do we also need to install grpc extension `pecl install grpc`  and `cat "extension=grpc.so" >> php.ini` here because these also seems like prequest for running php-grpc correctly? @stanley-cheung

Reply:
Several things
1. This file is to install prerequisites. We shouldn't be installing anything related to the actual grpc code here.
2. `sudo pecl install grpc` will go fetch the released version of the grpc php extension. That's not what you want here. You want to compile your new commited code from source.
3. Technically you don't have to put the `extension=grpc.so` line into a `php.ini` file. You can still add that at the command like `php -d extension=grpc.so ....`. Either way is fine.